scrapy_poet/middleware.py

```diff
@@ -39,20 +43,33 @@ def process_request(self, request: Request, spider: Spider):
     @inlineCallbacks
     def process_response(self, request: Request, response: Response,
                          spider: Spider):
-        """This method instantiates all ``Injectable`` sub-classes declared as
+        """This method instantiates all ``Injectable`` subclasses declared as
         request callback arguments and any other parameter with a provider for
         its type. Otherwise, this middleware doesn't populate
         ``request.cb_kwargs`` for this argument.
 
-        .. warning::
+        Currently, we are able to inject instances of the following
```

Comment:
Consider moving this docu section about what is injectable in providers to `PageObjectInputProvider` class or its `__init__` method. I think people writing their own providers would go there to seek for documentation.

Reply:
Done.